tools/src/test/java/org/apache/kafka/tools/consumer/group/DeleteConsumerGroupsTest.java

```diff
@@ -68,10 +66,6 @@
 @ExtendWith(value = ClusterTestExtensions.class)
 public class DeleteConsumerGroupsTest {
 
-    private static List<ClusterConfig> generator() {
```

Comment:
this method is executed dynamically. Maybe you can add comments to explain the use case?